src/NonStrippingCreditCard.php

```diff
@@ -28,4 +28,102 @@ public function setNumber($value)
          */
         return $this->setParameter('number', $value);
     }
+
+    /**
+     * Sets the payment instrument name.
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setPaymentInstrumentName($value)
+    {
+        /**
+         * @var static
+         */
```

Comment:
Why do you specify this as static?

Reply:
No particular reason, I was following suit of setNumber.

Reply:
Which already existed in this class.

Reply:
it's because setParameter is defined in omnipay common, but is not typed correctly. so we have to override the typehint here to help psalm out